diagnostic_feedback/templates/underscore/category.html

```diff
@@ -0,0 +1,13 @@
+<div class="category">
+    <input type="hidden" id="category[id][<%- order %>]" name="category[id][<%- order %>]" class="skip-validation" value="<%- id %>" />
+    <input placeholder="Name" type="text" id="category[name][<%- order %>]" name="category[name][<%- order %>]"  class="required" value="<%- name %>" />
+    <input placeholder="Image" type="text" id="category[image][<%- order %>]" name="category[image][<%- order %>]"  value="<%- image %>" />
+    <a href="#" data-tooltip="Delete" class="delete-category delete-button action-button">
+        <i class="icon fa fa-trash-o"></i>
+        <span class="sr">Delete</span>
```

Comment:
Because of where this trash can symbol appears in the UI, it looks to me like it will delete the "Image", not the whole category. Can you perhaps move it somewhere or make it visually more obvious that it deletes the whole category? Also the tooltip and screen reader text should say "Delete Category" to further make that more clear.